Description: Fix spinning wheel-of-death if no addressbooks exist for a user.
Origin:      https://github.com/nextcloud/contacts/pull/91
--- a/js/public/script.js
+++ b/js/public/script.js
@@ -227,6 +227,16 @@
 
 	AddressBookService.getAll().then(function(addressBooks) {
 		ctrl.addressBooks = addressBooks;
+		if (ctrl.addressBooks.length === 0) {
+			AddressBookService.create(t('contacts', 'Contacts')).then(function() {
+				AddressBookService.getAddressBook(t('contacts', 'Contacts')).then(function(addressBook) {
+					ctrl.addressBooks.push(addressBook);
+					ctrl.loading = false;
+					$scope.$apply();
+				});
+			});
+			return;
+		}
 		ctrl.loading = false;
 	});
 
